src/components/layout/PageContainer.tsx

```diff
@@ -1,22 +1,26 @@
-import { Typography } from '@mui/material';
-import { Box, Container, Stack } from '@mui/system';
+import { Container, ContainerProps, Stack } from '@mui/system';
 import { ReactNode } from 'react';
+import PageTitle from '@/components/layout/PageTitle';
 import { useIsMobile } from '@/hooks/useIsMobile';
 
 const PageContainer = ({
   children,
   title,
+  overlineText,
   actions,
+  maxWidth = 'lg',
 }: {
   children: ReactNode;
   title: ReactNode;
+  overlineText?: string;
   actions?: ReactNode;
+  maxWidth?: ContainerProps['maxWidth'];
 }) => {
   const isTiny = useIsMobile('sm');
   return (
     <Container
       component='main'
-      maxWidth='lg'
+      maxWidth={maxWidth}
```

Comment:
Reviewed with Ash at design standup - suggested just leaving this at `lg` so all the page containers match
```suggestion
      maxWidth='lg'
```